### ISSUE TITLE
`metrics-mysql.rb` should be renamed to indicate it is a handler

### ISSUE THREAD

**Issue by majormoses** [sensu-plugins/sensu-plugins-mysql#64]
Discovered in #63:

In order to help users quickly identify and start using various components we have a naming convention for files:
- `check-*`: indicates that it is a non metric check
- `metric-*`: indicates that it is a `metric check`
- `handler-*`: indicates that it is a handler
- `mutator-*`: indicates that it is a mutator 

I am not sure this is actually documented anywhere but it really needs to be (I will create another issue for that).

The file is named `metrics-mysql.rb` only after looking at the [code](https://github.com/sensu-plugins/sensu-plugins-mysql/blob/2.1.2/bin/metrics-mysql.rb#L26) did I realize it was not a metrics gatheror for mysql it was a handler to send metric data to mysql. To avoid confusion in a major release we should rename it to better reflect its nature, I am open to ideas but this is my initial thought:
- `handler-mysql-metrics.rb`
- `handler-mysql.rb`

@sensu-plugins/commit-bit any thoughts?

**Comment by eheydrick** [comment]
Naming is documented at https://github.com/sensu-plugins/documentation/blob/master/docs/developer_guidelines.md#naming-conventions. 👍 to renaming. `handler-mysql-metrics.rb` sounds good.

**Comment by majormoses** [comment]
So it is I actually opened that document and somehow skipped right over it. I am glad we do as I was very confused as I recalled seeing something. Great I will re-purpose the issue to expand on it a bit.
